src/content/en/updates/_shared/no-new-deprecations.html

```diff
@@ -0,0 +1 @@
+No features have been deprecated in this version of Chrome. Chrome Platform Status provides a <a href="https://www.chromestatus.com/features#browsers.chrome.status:%22Deprecated%22">list of deprecated features</a> from previous versions of Chrome.
```

Comment:
These should break at 80 chars, I'll merge for now, but please remember for next time.